packages/editor/src/components/page-attributes/parent.js

```diff
@@ -249,8 +256,12 @@ export function ParentRow() {
 							onClose={ onClose }
 						/>
 						<div>
-							{ __(
-								"Child pages inherit characteristics from their parent, such as URL structure. For instance, if 'Web Design' is a child of 'Services,' its URL would be mysite.com/services/web-design."
+							{ sprintf(
+								/* translators: %1$s The home URL of the WordPress installation without the scheme. */
+								__(
+									"Child pages inherit characteristics from their parent, such as URL structure. For instance, if 'Web Design' is a child of 'Services,' its URL would be %1$s/services/web-design."
```

Comment:
What's correct in American English: `'Services,'` or `'Services',`?

Reply:
Normally American English puts punctuation inside speech quotes as far as I know. 

However, in this example, we're citing a UI label. There's no comma in the label - adding one would change its meaning. 

I think it's therefore acceptable to use `'Services',`.